packages/appstore/src/modules/onboarding/onboarding.tsx

```diff
@@ -46,6 +47,20 @@ const Onboarding = ({ contents = trading_hub_contents }: TOnboardingProps) => {
 
     const onboarding_step = number_of_steps[step - 1];
 
+    const footer_header = contents[onboarding_step]?.footer_header;
+    const footer_text = contents[onboarding_step]?.footer_text;
+
+    const is_eu_footer_header = 'CFDs or Multipliers';
```

Comment:
@thisyahlen-deriv `is_...` variable is using for booleans , change the name to `eu_footer_header`, same for `is_eu_footer_text`